kaggle-classification/tf_trainer/tfrecord_input.py

```diff
@@ -0,0 +1,72 @@
+"""DatasetInput class based on TFRecord files."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+import nltk
+import functools
+import numpy as np
+from tf_trainer import dataset_input
+from tf_trainer import types
+from typing import Dict, Tuple, Callable, Iterator
+
+
+class TFRecordInput(dataset_input.DatasetInput):
+  """TFRecord based DatasetInput.
+
+  Handles parsing of TF Examples and tokenizing text (with nltk).
+  """
+
+  def __init__(self, train_path: types.Path, validate_path: types.Path,
+               text_feature: str, labels: Dict[str, tf.DType]) -> None:
+    nltk.download('punkt')
```

Comment:
I think it checks to see if it is up to date but if it is it doesn't download anything - so maybe you'd need a little bit of internet connection (but it might just proceed without internet too..). Will address if it actually becomes a problem.